testing/test_nodes.py

```diff
@@ -58,3 +58,30 @@ class FakeSession:
 
     outside = py.path.local("/outside")
     assert nodes._check_initialpaths_for_relpath(FakeSession, outside) is None
+
+
+def test_failure_with_changed_cwd(testdir):
+    """
+    Test failure lines should use absolute paths if cwd has changed since
+    invocation, so the path is correct (#6428).
+    """
+    p = testdir.makepyfile(
+        """
+        import os
+        import pytest
+
+        @pytest.fixture
+        def private_dir():
+            out_dir = 'ddd'
+            os.mkdir(out_dir)
+            old_dir = os.getcwd()
+            os.chdir(out_dir)
+            yield out_dir
+            os.chdir(old_dir)
```

Comment:
hmm I'm not so sure about this one, it seems like if the test restores the path properly then the absolute path would be incorrect / unnecesssary

Reply:
oh I see now, yeah nevermind this is fine 👍